parser/tests/integration/test_parse_unparse_integration.py

```diff
@@ -60,37 +62,30 @@ def test_round_trip(neo4j):
         # second tweak: renaming parent
         elif line.startswith("<fr:yaourts fruit de la passion"):
             line = "<en:Passion fruit yogurts"
+        # third tweak: removing unexisting parents
+        elif line.startswith("<en:milk"):
+            continue
         expected_lines.append(line)
 
     assert expected_lines == lines
 
 
-def test_two_branch_round_trip(neo4j):
```

Comment:
FYI: I removed this test because it was redundant with the test above and I didn't really see the point of it.

Reply:
The point was to verify we don't mess around when we have more than one project. (that we don't forget label in some part of the code).

So maybe we should instead keep this one.